### PR TITLE
fix url to coc page

### PR DESCRIPTION
**Pull request by reshamas**
## Ticket Description
No ticket associated with this issue.

## Description of Changes
On the "Contact Us" page, updated the URL to the CoC page.

## Before and After for UI Updates
No visible UI changes for this PR.
 
Before:

After:

## For PR Reviewer
- [ ] Does this file change the yarn.lock, package.json or package-lock.json file? If so, why?
- [ ] If this pr contains mobile and desktop changes, did you test on IphoneXr and desktop views?
- [ ] Does this file match the related tickets linked figma file, or does it pass the visual smell test?
- [ ] If this file contains javascript, does the javascript pass the smell test?
- [ ] If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

File: src/components/pages/ContactPage.jsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ function ContactUsFormComponent() {
             <label>
               <Field type="checkbox" name="toggle" />&nbsp;
               All communication must adhere to our&nbsp;
-              <a className={formStyleClasses.hyperlinks} href="/pages">
+              <a className={formStyleClasses.hyperlinks} href="/codeofconduct">
```
Comment (jtorreggiani) [inline src/components/pages/ContactPage.jsx:82]: @reshamas, we will want to use a React Router Link here.

You just need to update line 9 from 

```JavaScript
import { useNavigate } from 'react-router-dom';
```
to
```JavaScript
import { Link, useNavigate } from 'react-router-dom';
```
and update this line to

```JavaScript
<Link to="/codeofconduct" className={formStyleClasses.hyperlinks}>
  Code of Conduct
</Link>*
```

**Review by jtorreggiani — CHANGES_REQUESTED** [review]
@reshamas, just one comment to update this. Adding instructions on the fix. Let me know if you have any questions!

**Comment by reshamas** [comment]
> @reshamas, just one comment to update this. Adding instructions on the fix. Let me know if you have any questions!

Thanks for the review @jtorreggiani.  I submitted the updates.

**Review by jtorreggiani — APPROVED** [review]

**Comment by jtorreggiani** [comment]
@reshamas, looks like someone updated the same file on main which caused a merge conflict. I fixed the merge conflict for you and are going to merge this in.

**Review by jtorreggiani — APPROVED** [review]
LGTM ✅

**Pull request merged**